.storybook/preview.js

```diff
@@ -0,0 +1,10 @@
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
```

Comment:
you can put you GlobalStyles here so that it is integrated in story book
```js
export const decorators = [
  Story => <><GlobalStyles/><Story/></>
]
```